src/io/flutter/editor/WidgetIndentsHighlightingPass.java

```diff
@@ -336,7 +336,17 @@ public void paint(@NotNull Editor editor, @NotNull RangeHighlighter highlighter,
             // Draw horizontal line to the child.
             final VisualPosition widgetVisualPosition = editor.offsetToVisualPosition(childLine.getGuideOffset());
             final Point widgetPoint = editor.visualPositionToXY(widgetVisualPosition);
-            final int deltaX = widgetPoint.x - start.x;
+
+            final int startIdx = doc.getLineStartOffset(childLine.getLine());
```

Comment:
nit. Here and elsewhere, abbreviations so write
startIndex rather than startIdx.

Reply:
add a comment explaining why this code is needed as well as tracking widgetPoint. You could reference the bug and talk about when the code has changed but we don't yet have a new outline.
there is an additional issues that we are just working around that there seems to be a race condition that @devoncarew is able to reproduce where the invalid offsets sometimes stay around even after a new outline is sent.

Reply:
Should I remove referring to #4297 for this change? Because this change is related but it doesn't address getting invalid offsets in the new outline.